includes/functions.php

```diff
@@ -1463,3 +1463,73 @@ function report_this($message) {
     return '<h2>'.$message.' Please <a href="'.$config['project_issues'].'">report this</a> to the '.$config['project_name'].' developers.</h2>';
 
 }//end report_this()
+
+function hytera_h2f($number,$nd)
```

Comment:
Does this function just change hex to float?

Could you just use floatval($hex) or (float)$hex?